sound/soc/sof/ops.h

```diff
@@ -330,7 +345,8 @@ snd_sof_pcm_platform_pointer(struct snd_sof_dev *sdev,
 	if (sof_ops(sdev) && sof_ops(sdev)->pcm_pointer)
 		return sof_ops(sdev)->pcm_pointer(sdev, substream);
 
-	return 0;
+	dev_err(sdev->dev, "error: %s not defined\n", __func__);
+	return -ENOTSUPP;
```

Comment:
Are PCM operations actually required? Take the example of Sue Creek, you may have all the processing done on the device itself with no PCM being passed back to the host?

Reply:
@plbossart ok i will make these optional